docs/making-release-notes.md

```diff
@@ -0,0 +1,38 @@
+## Making release notes
+This documents the process for making release notes for a release.
+
+### 1) Note the PR number of the previous release
+Find the PR that was merged with the previous release.  Remember this number
+
+### 2) Build the release-notes tool
```

Comment:
numbers 2/3 here are redundant with the README.md file and one or the other is likely to get out of date (there is already a typo in the README that isn't a typo here).